docs/response/index.html

```diff
@@ -0,0 +1,30 @@
+.. _xrtpy-response:
+
+***************************
+Response (`xrtpy.response`)
+***************************
+
+.. py:module:: xrtpy.response
+.. currentmodule:: xrtpy.response
+
+Introduction
+============
+
+The `xrtpy.response` subpackage includes functionality for describing
+XRT_ channels & filters, finding the effective area, and calculating the
+temperature response function.
```

Comment:
This is where the narrative documentation for `xrtpy.response` will need to be added.